content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -299,6 +299,24 @@ This annotation is part of the Kubernetes Resource Model (KRM) Functions Specifi
 which is used by Kustomize and similar third-party tools.
 For example, Kustomize removes objects with this annotation from its final build output.
 
+
+### container.apparmor.security.beta.kubernetes.io/* (beta) {#container-apparmor-security-beta-kubernetes-io}
+
+Type: Annotation
+
+Example: `container.apparmor.security.beta.kubernetes.io/my-container: my-custom-profile`
+
+Used on: Pods
+
+This annotation allows you to specify the AppArmor security profile for a container within a
+Kubernetes pod. 
+To learn more, see the [AppArmor](/docs/tutorials/security/apparmor/) tutorial.
+The tutorial illustrates using AppArmor to restrict a container's abilities and access.
+
+By setting this annotation, you can define a custom AppArmor profile for a container,
+which dictates the set of rules and restrictions that the containerized process must
```

Comment:
Please try avoid repeating ourselves when possible:

```suggestion
The profile specified dictates the set of rules and restrictions that the containerized process must
```

Reply:
Thanks for the suggestion @tengqm. I'll make sure not to use from the next time.